### PR TITLE
2.11.1 Carbon fails as nested module

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,172 @@
+
+# 2.12.1
+
+* Fixed: Carbon fails to import multiple times in the same session.
+* Fixed: Carbon fails when used as a nested module and Carbon is loaded globally or by nested in another module.
+
+
+# 2.12.0
```
Comment (DecoyJoe) [inline CHANGELOG.md:8]: Version headings are incorrect, this should be `2.11.1`

File: Carbon/Carbon.psm1 [hunk 2]
```diff
@@ -94,6 +117,86 @@ $TrustedHostsPath = 'WSMan:\localhost\Client\TrustedHosts'
 Write-Timing ('Adding System.DirectoryServices.AccountManagement assembly.')
 Add-Type -AssemblyName 'System.DirectoryServices.AccountManagement'
 
+$MyInvocation.MyCommand.ScriptBlock.Module.OnRemove = { Write-Host 'I am being removed!' }
```
Comment (DecoyJoe) [inline Carbon/Carbon.psm1:120]: Did you intend to leave this here?

**Review by DecoyJoe — CHANGES_REQUESTED** [review]

File: Carbon/Carbon.psm1 (hunk 2 above)
Comment (splatteredbits) [reply Carbon/Carbon.psm1]: Nope. Removed.

File: CHANGELOG.md (hunk 1 above)
Comment (splatteredbits) [reply CHANGELOG.md]: Yeah. There was some versioning confusion. 😆

**Review by DecoyJoe — APPROVED** [review]

**Pull request merged**